helmclient.go

```diff
@@ -272,10 +272,11 @@ func (c *Client) EnsureTillerInstalled(ctx context.Context) error {
 	}
 
 	i := &installer.Options{
-		ImageSpec:      c.tillerImage,
-		MaxHistory:     defaultMaxHistory,
-		Namespace:      c.tillerNamespace,
-		ServiceAccount: tillerPodName,
+		AutoMountServiceAccountToken: true,
```

Comment:
The service account was defined before but now we need to auto mount it. This changed in Helm 2.9.1.

https://github.com/helm/helm/pull/3784